Feature/flex actions

j-ferber opened the pull request:
Added the flex style option for the take action block, allowing the last row of actions to be centered if not full. Added the types for the take action block and the team gallery block as well. Formatting changes came from npm run format.

Grid layout example: 
![image](https://github.com/stevensblueprint/inreach-website/assets/119906373/90238850-7d64-41d6-9137-ed95dc39626f)

Flex layout example:
![image](https://github.com/stevensblueprint/inreach-website/assets/119906373/1f860e57-5428-4237-abf9-34105aab34aa)

Display selector:
![image](https://github.com/stevensblueprint/inreach-website/assets/119906373/3874b4a3-6828-47f4-b92f-f51770048d62)

You can view the changes at http://localhost:3000/admin/index.html#/~/example-actions by adding another take action display section.

j-ferber committed:
Added the flex option to the TakeAction block. Added types as well.

j-ferber committed:
Adjusted the grid layout to be the same size as the flex layout.

ShawnAviles commented on src/components/blocks/TakeAction.tsx:
```diff
@@ -1,98 +1,155 @@
+	return display === 'flex' ? (
+		<div className='md:max-w-[33.333333%] md:w-1/3 p-1 flex flex-grow w-full'>
+			<div className='w-full gap-3 border-2 border-black rounded-lg flex flex-col justify-between items-center text-black p-4 bg-white'>
+				{data?.title && <h1 className='lg:text-2xl md:text-xl text-2xl font-bold'>{data.title}</h1>}
```
The h1 title tag should include `text-center` to account for titles that span more than one line. As of rn, it becomes left-aligned when too long.

ShawnAviles commented on src/components/blocks/TakeAction.tsx:
```diff
@@ -1,98 +1,155 @@
+		>
+			{data.items &&
+				data.items.map((block: PageBlocksActionsItems | null, i: number) => {
+					if (block) return <TakeAction data={block} key={i} display={data.display as string} />
```
Any way we can address the keys prop here? Shouldn't rely on the index only. Especially since we are able to change the order around for the cards

ShawnAviles reviewed (changes requested):
Looks good. Made small comments on the styling of the header text and the unique keys that Joe had mentioned.

j-ferber committed:
Edited the README.md  (#49)

Changed the README.md so it included the steps to adding and viewing
your blocks using the CMS

j-ferber committed:
Added the different fits for actions blocks

j-ferber committed:
Merge branch 'main' into feature/flexActions

ShawnAviles reviewed (approved):
Ty for addressing the comments in the PR. Looks good! 👍
